ams/server/src/main/java/com/netease/arctic/server/table/TableConfiguration.java

```diff
@@ -16,9 +16,11 @@ public class TableConfiguration {
   private long changeDataTTLMinutes;
   private boolean cleanOrphanEnabled;
   private long orphanExistingMinutes;
+  private boolean autoCreateTagEnabled;
```

Comment:
How about putting the `autoCreateTagEnabled` into the `TagConfiguration`?

Reply:
It is an oversight. I have modified it.